cli/internal/cmd/upgradeplan.go

```diff
@@ -186,7 +186,7 @@ func getCompatibleImageMeasurements(ctx context.Context, client *http.Client, pu
 			return err
 		}
 
-		if err := img.Measurements.FetchAndVerify(ctx, client, measurementsURL, signatureURL, pubK); err != nil {
+		if _, err := img.Measurements.FetchAndVerify(ctx, client, measurementsURL, signatureURL, pubK); err != nil {
```

Comment:
From my understanding we also want the rekor verification here at some point, right? Or is there something preventing us?

Reply:
Yes 👍  added it!